packages/flutter/lib/src/rendering/semantics.dart

```diff
@@ -149,6 +158,8 @@ class SemanticsData {
     }
     if (label.isNotEmpty)
       buffer.write('; "$label"');
+    if (textDirection != null)
+      buffer.write('; $textDirection');
     buffer.write(')');
```

Comment:
No cool diagnostics node here!

Reply:
Yeah, @jacob314 didn't convert the semantics logic to use the new diagnostics logic. And @goderbauer has since made it much harder to do so by introducing a way to control what order they're painted in. It's a whole thing. ;-)

Reply:
I'll go ahead and work on converting the Semantics logic now.I don't expect being able to display the data in two orders should be that big an issue to support.

Reply:
Oh sweet, thanks!